server/meshmodel/aws-cloudfront-controller/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,76 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+    "name": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+  },
+  "model": {
+    "name": "aws-cloudfront-controller",
+    "version": "*",
+    "displayName": "AWS CloudFront",
+    "category": {
+      "name": "App Definition and Development",
+      "metadata": null
+    },
+    "metadata": {
+      "isAnnotation": false,
+      "published": false,
+      "subCategory": "Service Proxy",
+      "svgColor": "",
+      "svgComplete": "",
+      "svgWhite": ""
+    }
+  },
+  "subType": "Inventory",
+  "evaluationQuery": "heirarchical_inventory_relationship",
+  "selectors": [
+    {
+      "allow": {
+        "from": [
+          {
+            "kind": "Pod",
```

Comment:
@MUzairS15 Since Field Export will have relationships with most Kubernetes components, instead of manually listing them all one by one, can we use '' in the 'kind' field like this: `"kind": "*"`?

Reply:
Yes. :thu

Reply:
Great. Is help needed to insert `*`? Or is there something else that this PR is awaiting?

Reply:
@pandeyshubham03 help, please.

Reply:
Instead of adding individual components in the `from` section,
We can add:
```sh
{
            "kind": "*",
            "model": "Kubernetes",
            "patch": {
              "mutatorRef": [["name"], ["namespace"], ["type"]],
              "description": ""
            }
}
```
This will establish the relationship from all Kubernetes components to the desired component

Reply:
I know. Which is why I think it’s ridiculous that this has taken so long and that neither one of will make the extremely simple change that this requires to finish the PR. But instead I have to sit here and review the PR many times over.